Src/FluentAssertions/TypeDescriptionUtility.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace FluentAssertions
+{
+    internal static class TypeDescriptionUtility
+    {
+        public static string GetDescriptionOfObjectType(object obj)
+        {
+            return (obj is null) ? "<null>" : GetTypeDescription(obj.GetType());
+        }
+
+        public static string GetTypeDescription(Type type)
```

Comment:
🔧 This should be `private` since it's an implementation detail of the other method.

Reply:
I didn't intend for it to be an implementation detail. Presently, the only caller into `TypeDescriptionUtility` is passing in object references, but if in the future the need to describe a `Type` arises, this method is intended to be called directly. If I make it `private`, then it won't be discoverable.

Reply:
That's exactly why we prefer to keep things private. There's no need to make it generic, so then we don't.

Reply:
Very well, `private` it is. :-P